lib/workers/repository/config-migration/branch/commit-message.ts

```diff
@@ -4,43 +4,60 @@ import type { CommitMessage } from '../../model/commit-message';
 import { CommitMessageFactory } from '../../model/commit-message-factory';
 
 export class ConfigMigrationCommitMessageFactory {
-  private readonly config: RenovateConfig;
+  private commitMessage: string | null = null;
+  private prTitle: string | null = null;
 
-  private readonly configFile: string;
+  constructor(
+    private readonly config: RenovateConfig,
+    private readonly configFile: string
+  ) {}
 
-  constructor(config: RenovateConfig, configFile: string) {
-    this.config = config;
-    this.configFile = configFile;
-  }
-
-  create(): CommitMessage {
+  private create(messageTopic: string): CommitMessage {
     const { commitMessage } = this.config;
+    let { commitMessageAction, commitMessageTopic } = this.config;
 
-    this.config.commitMessageAction =
-      this.config.commitMessageAction === 'Update'
-        ? ''
-        : this.config.commitMessageAction;
+    if (commitMessageAction === 'Update') {
```

Comment:
This type of code indicates to me that we've overcomplicated this commit message